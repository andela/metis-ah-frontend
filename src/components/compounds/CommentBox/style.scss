#comment-box {
  padding: 10px;
  margin-top: 50px;
  border-radius: 4px;
}
