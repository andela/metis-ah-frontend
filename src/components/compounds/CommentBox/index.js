import React from 'react';
// import PropTypes from 'prop-types';
import './style.scss';

import AddComment from 'Components/compounds/AddComment';
import CommentsView from 'Components/compounds/CommentsView';

export class CommentBox extends React.Component {
  state = {};

  render() {
    return (
      <div id="comment-box">
        <AddComment />
        <CommentsView />
      </div>
    );
  }
}

// CommentBox.propTypes = {};

export default CommentBox;
