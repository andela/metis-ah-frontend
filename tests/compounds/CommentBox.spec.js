import React from 'react';
import { shallow } from 'enzyme';

import CommentBox from 'Components/compounds/CommentBox'


describe('CommentBox Tests', () => {
  test('it renders successfully', () => {
    const wrapper = shallow(<CommentBox />);
    expect(wrapper.exists()).toBe(true);
  });

  test('it matches snapshot', () => {
    const wrapper = shallow(<CommentBox />);
    expect(wrapper).toMatchSnapshot();
  });
});
