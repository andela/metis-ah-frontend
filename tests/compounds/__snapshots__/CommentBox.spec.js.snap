// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommentBox Tests it matches snapshot 1`] = `
<div
  id="comment-box"
>
  <withRouter(Connect(AddComment)) />
  <withRouter(Connect(CommentsView)) />
</div>
`;
